pyof/v0x01/common/utils.py

```diff
@@ -90,24 +86,24 @@ def new_message_from_message_type(message_type):
 
 
 def new_message_from_header(header):
-    """Method that receives a OpenFlowHeader and then returns an empty message
-    regarding the message_type attribute from the header.
+    """Given an OF Header, return an empty message of header's message_type.
 
     Args:
-        header (Header): Unpacked OpenFlow Header
+        header (Header): Unpacked OpenFlow Header.
 
     Returns:
-        empty OpenFlow message of the same type of message_type attribute from
-        the given header. The header attribute of the message will be populated
+        Empty OpenFlow message of the same type of message_type attribute from
+        the given header.
+        The header attribute of the message will be populated.
 
     Raises:
-        KytosUndefinedMessageType: unkown Message_Type
+        KytosUndefinedMessageType: Unkown Message_Type.
     """
     message_type = header.message_type
     if not isinstance(message_type, Type):
         try:
             if isinstance(message_type, str):
-                message_type = Type[message_type]
```

Comment:
Probably you introduced a bug. message_type here is a string:
```python3
message_type = Type("OFPT_HELLO")
ValueError: 'OFPT_HELLO' is not a valid Type
```

Please check this with @diraol to make sure that this way is right.